velox/exec/tests/AggregationTest.cpp

```diff
@@ -3671,4 +3671,27 @@ TEST_F(AggregationTest, ignoreNullKeys) {
   AssertQueryBuilder(makePlan(true)).assertEmptyResults();
 }
 
+TEST_F(AggregationTest, aggWithOrderBy) {
```

Comment:
aggWithOrderBy test name suggests that this test verifies aggregation order sorted inputs, however, the test actually verified that ORDER BY clause is ignored when aggregate function is not order-sensitive.

Consider renaming for clarify. Perhaps, ignoreOrderBy.

// Verify that ORDER BY clause is ignored for aggregates that are not order sensitive.
TEST_F(AggregationTest, ignoreOrderBy) {

Reply:
Done.